storage/test/storageTest.php

```diff
@@ -757,6 +757,49 @@ public function testChangeDefaultStorageClass()
         );
     }
 
+    public function testGetBucketWithAutoclass()
+    {
+        $bucketName = uniqid('samples-create-autoclass-');
+        $bucket = self::$storage->createBucket($bucketName, [
+            'autoclass' => [
+                'enabled' => true,
+            ],
+            'location' => 'US',
+        ]);
+
+        $output = self::runFunctionSnippet('get_bucket_autoclass', [
+            $bucketName,
+        ]);
+        $bucket->delete();
+
+        $this->assertStringContainsString(
+            sprintf('Bucket %s has autoclass enabled: %s', $bucketName, true),
+            $output
+        );
+    }
+
+    public function testSetBucketWithAutoclass()
+    {
+        $bucket = self::$storage->createBucket(uniqid('samples-set-autoclass-'));
+        $info = $bucket->reload();
+        $this->assertArrayNotHasKey('autoclass', $info);
+
+        $output = self::runFunctionSnippet('set_bucket_autoclass', [
+            $bucket->name(),
+            true
+        ]);
+        $bucket->delete();
+
+        $this->assertStringContainsString(
+            sprintf(
+                'Updated bucket %s with autoclass enabled',
+                $bucket->name(),
+                true
```

Comment:
I think the `true` is here by accident
```suggestion
                $bucket->name()
```